src/main/java/frc/robot/Constants.java

```diff
@@ -21,23 +22,28 @@
     
     /* HID Constants */
     
-    //Set's the name for configureJoysticks() in RobotContainer.java to search for
-    public static String driveJoystickName = "Logitech Dual Action";
+    //Set's the name for configureJoysticks() in RobotContainer.java to search for.
+    public static String driveJoystickName = "Wireless Controller";
     public static String auxJoystickName = "Controller (HORIPAD S)";
 
     //Joystick Axis Values
-    public static int driveJoystickRotationAxisNum = 2; //The Logitech Dual Action controller's right-hand-side joystick does not correctly 
+    public static int driveJoystickRotationAxisNum = 2; //The joystick's controller's right-hand-side joystick does not correctly.
+    public static double driveJoystickTurnCoefficient = 0.25;
+
+    //Joystick Button Values
+    public static int auxAButton = 1; //Represents the A button on the Horipad Gamecube controller.
+    public static int auxXButton = 3; //Represents the X button on the Horipad Gamecube controller.
 
     /* Drive Constants */
 
     //Talon SRX ID's
     public static final double driveDeadband = 0.01; //1% Deadband.
-    public static int[] rightTalonID = new int[]{1, 2}; //Encoder on 1.
-    public static int[] leftTalonID = new int[]{3, 4}; //Encoder on 3.
+    public static int[] rightTalonID = new int[]{2, 3}; //Encoder on 1.
```

Comment:
I'm just eyeballing this here, but you might want to abstract the representation of "talon id" into a separate class - maybe not, it really depends on you're usage of how, and how the ID is consumed

Reply:
The Talon ID's are just passed to the WPI_TalonSRX constructor for instantiation. They're in the array as a representation that the two sides are grouped together